src/fosslight_dependency/_package_manager.py

```
@@ -87,122 +88,203 @@ def parse_direct_dependencies(self):
         try:
-            with open(const.SUPPORT_PACKAE.get(self.package_manager_name), 'r', encoding='utf-8') as original:
-                data = original.read()
-            with open(const.SUPPORT_PACKAE.get(self.package_manager_name), 'w', encoding='utf-8') as modified:
-                modified.write(f"{build_script}\n{data}")
-            ret = True
+            with open(gradle_file, 'r', encoding='utf-8') as f:
+                data = f.read()
+            with open(gradle_file, 'w', encoding='utf-8') as f:
+                f.write(f"{build_script}\n{data}")
+            return True
         except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)